### ISSUE TITLE
Instalar OpenCv en Python

### ISSUE THREAD

**Issue by LeonardoEnriquezA** [LeonardoEnriquezA/IA#1]
**Librerias requeridas en Python**
_import imutils_
from PIL import Image
_import argparse_
import ast
_import json_
import requests
import numpy as np
from sklearn.decomposition import PCA
from sklearn.mixture import GaussianMixture
from sklearn.externals import joblib
from sklearn import preprocessing
_import pyodbc_
import urllib.request
import threading
from imutils import build_montages
from multiprocessing.dummy import Pool
import cv2, os
from skimage.measure import compare_ssim
from random import shuffle
from shutil import copyfile
